Sources/Tracing/TracingTime.swift

```diff
@@ -23,10 +23,16 @@ import Glibc
 import Android
```

Comment:
Note, there is a lot of existing swiftformat lint in this repository, including the existing indentation on many of these import blocks. I fixed the lint on the lines of code I'm changing in this PR. But I didn't fix it for everything else, to avoid making the diff far too large.